Makefile

```diff
@@ -32,6 +32,7 @@ build-no-default:
 	PKG_CONFIG_ALLOW_CROSS=1 \
 	RUSTFLAGS='-D warnings' \
 	cargo build --no-default-features --target $(TARGET)
+	! (ldd ./target/${TARGET}/debug/stratisd | grep libdbus)
```

Comment:
I think you have the path incorrect...

`! (ldd ./target/${TARGET}/stratisd | grep libdbus)`

Reply:
Guess this isn't it...